reactor-core/src/main/java/reactor/core/publisher/Operators.java

```diff
@@ -2576,6 +2577,27 @@ private LiftFunction(@Nullable Predicate<Publisher> filter,
 				return new FluxLift<>(publisher, lifter);
 			}
 		}
+
+		static class NamedLifter<T, U, R> implements BiFunction<T, U, R> {
```

Comment:
could be removed by putting the `name` field directly in `LiftFunction` then passing it to the various `xxxLift` constructors, wdyt?

alternatively, these constructors could even take the `LiftFunction` itself I think... I'd need to double check there is any case where such an operator can be created with something else that the `LiftFunction`.

Reply:
done